core/trino-main/pom.xml

```diff
@@ -415,6 +415,12 @@
             <groupId>io.airlift</groupId>
             <artifactId>jaxrs-testing</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
```

Comment:
comment why needed and move exclusion to airlift if this is problematic and not needed transitive dep

Reply:
There are two hamcrest versions on classpath with different GAV so I cannot settle a single version: [org.hamcrest:hamcrest-core:1.3, org.hamcrest:hamcrest:2.2]